client/src/pages/Signin.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Signin = ({AuthModal}) => {
+    return (
+        <>
+            {AuthModal}
+        </>
```

Comment:
Unless you're going to return more JSX components at the same level as `AuthModal`, the `fragments` are superfluous.
Check out the documentation on [Fragments](https://reactjs.org/docs/fragments.html).